test_rigs/unit-tests/configs/juniper_route_filter

```diff
@@ -0,0 +1,16 @@
+set system host-name juniper_route_filter
```

Comment:
Minor: please add `#` line at top of file to match style of other unit tests. This will affect refs, so do it before generating new `.ref.testout` files.